testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -35,9 +35,10 @@
 
 import MDAnalysis as mda
 import MDAnalysis.core.topologyattrs as tpattrs
-from MDAnalysis.core import groups
+from MDAnalysis.core._get_readers import get_reader_for
 from MDAnalysis.core.topology import Topology
 from MDAnalysis.exceptions import NoDataError
+from soupsieve import escape
```

Comment:
Why is this here? Can't seem to see where this is being used.

Reply:
Ahh damn. sorry